hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/props/NodeConnectInfo.java

```diff
@@ -52,11 +54,15 @@ public NodeConnectInfo(String inString) {
         } else {
             port = DEFAULT_PORT;
         }
+
+        tlsPort = DEFAULT_TLS_PORT;
```

Comment:
Don't you still need this
```
if (ports.length > 1) {
            tlsPort = ports[1];
        } else {
            tlsPort = DEFAULT_TLS_PORT;
        }

Reply:
Reverted changes in `NodeConnectInfo` as @tinker-michaelj suggested since it's usage is not recommended going forward.